Documentations/set_manufacturing_data.py

```
@@ -381,7 +464,29 @@ def main():
-            set_bluetooth_name(ser)
+            try:
+                # First try in the current session (often still in service mode)
+                set_bluetooth_name(ser)
+            except Exception as e:
+                if needs_reinit_for_bt_name and "not echoed" in str(e):
+                    print(f"Warning: {e}. Bluetooth disconnected after 0x80.")
+                    print("Please reconnect /dev/rfcomm0 now (e.g. via rfcomm), then press Enter.")
+                    try:
+                        ser.close()
+                    except Exception:
+                        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)